### ISSUE TITLE
Enhancement: update all A records on domain

### ISSUE THREAD

**Issue by osoviejo** [leandrokoiti/GoDaddyDnsUpdater#2]
Currently it only updates the domain A record, but not the subdomains.

**Comment by leandrokoiti** [comment]
Hey @osoviejo , about this feature, you mean automatically update every subdomain of a given domain with the same IP?

**Comment by osoviejo** [comment]
Yes.  If the blanket update is concerning at all (although I'm not sure why it would be--any host records using the old IP are broken until updated), it could be an option in Settings (or for the glitzy version there could be a checkbox for each host record found and listed).

But really, I can't come up with a scenario where one wouldn't want all the host records with the old IP updated.  So I'd just go simple and do the whole batch without a bunch of config work.  If someone comes forward with a good use case for more granular control later, it can always be evaluated then.

**Comment by ColossusFX** [comment]
Not sure if your still active on this.

I would like to only update an IP for only 1 subdomain A record.

Also this doesn't seem to work anymore.

No domains are listed.

I didn't want to have to re-write a new app just to update one A record!

**Comment by leandrokoiti** [comment]
Hey @osoviejo 
I still didn't have time to look further into this feature but I believe it's not complicated so as soon as I merge @Saad-Alothman's version and update the version with a few little bugs I fixed, I believe I will be able to update with your request
Since not everyone would want to have every subdomain updated at once (at least in my case, I direct each subdomain to a different IP some times), I will add an option to enable/disable this feature as well, sorry for taking so long, been a lot busy lately just found time to check the code at this moment
